R/susie.R

```diff
@@ -522,7 +522,7 @@ susie_rss <- function(z = NULL, R, n = NULL,
     estimate_prior_variance, estimate_prior_method,
     unmappable_effects, check_null_threshold, prior_tol,
     residual_variance_lowerbound, residual_variance_upperbound,
-    model_init = s_init, coverage, min_abs_corr,
+    model_init = model_init, coverage, min_abs_corr,
```

Comment:
I think it is prefered to use positional arguments followed by optional arguments `=`, not a mix use of them.